seg_lapa/config_parse/callbacks_conf.py

```diff
@@ -40,15 +41,25 @@ def get_callback(self):
         return EarlyStopping(**args_dict)
 
 
+@dataclass(frozen=True)
+class CheckpointConf:
+    """Dataclass just to initialize and return the Checkpoint Callback"""
+
+    def get_callback(self, logs_dir):
+        checkpoint_callback = ModelCheckpoint(dirpath=logs_dir)
+        return checkpoint_callback
+
+
 @dataclass(frozen=True)
 class StandardCallbacksConf(CallbacksConf):
     """Get a dictionary of all the callbacks."""
 
     early_stopping: Dict
```

Comment:
Once we add the model checkpoints vales to .yaml file, we'll need to add an entry here too.